operators/pkg/custom-metrics-server/metrics_server.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package metricsserver;
+option go_package = "./custom_metrics_server";
+
+service MetricsServer {
+    // ContainerMetrics returns metrics of the "application container" related to the required PodName. 
+    // If the container does not exist, the call returns an error.
+    rpc ContainerMetrics(ContainerMetricsRequest) returns (ContainerMetricsResponse) {}
+}
+
+message ContainerMetricsResponse {
+    float cpu_perc = 1; 
+	uint64 mem_bytes = 2;
+	uint64 disk_bytes = 3;
+}
+
+message ContainerMetricsRequest {
+    // Filter needed to find target "application container"
+    string pod_name = 1;
+}
```

Comment:
verify that all files are terminated with a blank line, git isn't usually really glad to work with files that aren't